nsxt/resource_nsxt_manager_cluster.go

```diff
@@ -175,75 +132,58 @@ func getClusterInfoFromHostNode(d *schema.ResourceData, m interface{}) (string,
 	min := c.CommonConfig.MinRetryInterval
 	max := c.CommonConfig.MaxRetryInterval
 	maxRetries := c.CommonConfig.MaxRetries
-	hostIP := ""
+	hostIPs := []string{}
 	for i := 0; i < maxRetries; i++ {
 		clusterConfig, err := client.Get()
 		if err != nil {
-			return "", "", "", handleReadError(d, "Cluster Config", "", err)
+			return "", "", hostIPs, err
```

Comment:
if this fails after the first iteration we might return a non empty host IP list.
Is that expected?

Reply:
Should be ok since error is always checked, and other return values are disregarded when error is present